packages/lib-classifier/src/plugins/tasks/survey/models/SurveyTask.js

```diff
@@ -41,9 +41,33 @@ const Survey = types.model('Survey', {
   questions: types.frozen({}),
   questionsMap: types.frozen({}),
   questionsOrder: types.array(types.string),
-  thumbnails: types.maybe(types.string),
+  thumbnails: types.maybe(
+    types.enumeration('thumbnails', ['show', 'hide', 'default'])
+  ),
   type: types.literal('survey')
 })
+  .preProcessSnapshot(snapshot => {
+    if (snapshot.hasOwnProperty('alwaysShowThumbnails')) {
+      const newSnapshot = { ...snapshot }
+      // 'true' and 'false' are both true.
+      if (snapshot.alwaysShowThumbnails === 'false') {
+        newSnapshot.alwaysShowThumbnails = false
+      }
+      if (snapshot.alwaysShowThumbnails === 'true') {
+        newSnapshot.alwaysShowThumbnails = true
+      }
+      newSnapshot.alwaysShowThumbnails = !!newSnapshot.alwaysShowThumbnails
```

Comment:
I think this line fixes the empty string issue 👍 .

Reply:
Yeah. String truthiness is determined by `string.length` so empty strings are false (and `'false'` is true.)